vcl_templates/www.vcl.erb

```diff
@@ -327,6 +323,12 @@ sub vcl_fetch {
       set beresp.ttl = 900s;
       set beresp.http.Cache-Control = "max-age=900";
     }
+
+    # Only cache non-GET/HEAD requests if they return a 404 or 405
+    if (!(req.request == "GET" || req.request == "HEAD") && !http_status_matches(beresp.status, "404,405")) {
+      set beresp.ttl = 0s;
+      set beresp.cacheable = false;
```

Comment:
If you're using these two to avoid caching I think you can do `return(pass)`: https://varnish-cache.org/docs/2.1/tutorial/vcl.html#actions

Reply:
Thanks, I had to do some reading to understand why, but that makes sense.

Is it correct that the resulting behaviour of calling `return(pass)` in `vcl_fetch` would be to create a hit-for-pass object, and thereby avoid a `lookup` for this request in future (while the ttl lasts)?